code/datums/sprite_accessories.dm

```diff
@@ -1914,11 +1905,12 @@
 	icon_state = "round"
 
 /datum/sprite_accessory/ears/fox
+	// This shouldn't evem be in the cat ear accessory list at all,
+	// but bodypart overlays don't support that yet
```

Comment:
CC @Time-Green 
I'd like to make the `ears_list` JUST a list of cat ears, but in testing we don't seem to have a good way to make `/datum/bodypart_overlay/mutant`s for a single sprite overlay
(IE: `/datum/bodypart_overlay/mutant/ears/fox` which JUST does `/datum/sprite_accessory/ears/fox`)
Am I missing something?

Reply:
You can set sprite_datum and then set imprint_on_next_insertion to FALSE

Reply:
Oh I misunderstood. Honestly just rename it to` /datum/sprite_accessory/fox_ears`. It's cringe but sprice_accessory is already hopeless imo